slurm_ops_manager/mpi.py

```diff
@@ -0,0 +1,49 @@
+"""MPI Class."""
```

Comment:
This should either be a complete sentence, or it should not have a period.

Reply:
Without a period the lint fails:

`slurm_ops_manager/mpi.py:1:1: D400 First line should end with a period`

Reply:
Can you rebase on top of the current `master`? We just added a CI to lint the code on pull requests.

Could you improve a bit this docstring to explain what is the purpose of the class?

Reply:
Done